CadRevealFbxProvider.Tests/FbxProviderTests.cs

```diff
@@ -58,13 +60,50 @@ public void FbxImporterLoadFileTest()
         Iterate(rootNode);
     }
 
+    [Test]
+    public void GreenAndRedCube_LoadFile_VerifyCorrectColors()
```

Comment:
Consider adding another test with the GeometryOffset if we are able to create a file with it?